docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc

```diff
@@ -0,0 +1,239 @@
+
+[[release-notes]]
+= Release notes
+
+This section summarizes the changes in each release.
+
+* <<release-notes-8.6.0>>
+
+Also see:
+
+* {kibana-ref}/release-notes.html[{kib} release notes]
+* {beats-ref}/release-notes.html[{beats} release notes]
+
+// begin 8.6.0 relnotes
+
+[[release-notes-8.6.0]]
+== {fleet} and {agent} 8.6.0
+
+Review important information about the {fleet} and {agent} 8.6.0 release.
+
+[discrete]
+[[breaking-changes-8.6.0]]
+=== Breaking changes
+
+Breaking changes can prevent your application from optimal operation and
+performance. Before you upgrade, review the breaking changes, then mitigate the
+impact to your application.
+
+[discrete]
+[[breaking-1994]]
+.Each input in an agent policy must have a unique ID 
+[%collapsible]
+====
+*Details* +
+Each input in an agent policy must have a unique ID, like `id: my-unique-input-id`.
+This change only affects standalone agents. Unique IDs are automatically generated in
+agent policies managed by {fleet}. For more information, refer to
+{agent-pull}/1994[#1994]
+
+*Impact* +
+Make sure that your standalone agent policies have a unique ID.
+====
+
+[discrete]
+[[breaking-1140]]
+.Diagnostics `--pprof` argument has been removed and is now always provided
+[%collapsible]
+====
+*Details* +
+The `diagnostics` command gathers diagnostic information about the {agent} and
+each component/unit it runs. Starting in 8.6.0, the `--pprof`
+argument is no longer available because `pprof` information is now always
+provided. For more information, refer to {agent-pull}1140[#1140].
+
+*Impact* +
+Remove the `--pprof` argument from any scripts or commands you use.
+====
+
+[discrete]
+[[known-issues-8.6.0]]
+=== Known issues
+
+[discrete]
+[[known-issue-issue-2066]]
+.Osquery live query results can take up to five minutes to show up in {kib}
+[%collapsible]
+====
+*Details* +
+A known issue in {agent} may prevent live query results from being available
+in the {kib} UI even though the results have been successfully sent to {es}. 
+For more information, refer to {agent-issue}2066[#2066].
+
+*Impact* +
+Be aware that the live query results shown in {kib} may be stale.
+====
+
+[discrete]
+[[new-features-8.6.0]]
+=== New features
+
+The 8.6.0 release adds the following new and notable features.
+
+{fleet}::
+* Differentiate kubernetes integration multipage experience {kibana-pull}145224[#145224]
+* Add prerelease toggle to Integrations list {kibana-pull}143853[#143853]
+* Add link to allow users to skip multistep add integration workflow {kibana-pull}143279[#143279]
+* Add new action for single agents to request diagnostics {kibana-pull}142369[#142369]
+
+{agent}::
+* Upgrade Node to version 18.12.0 {agent-pull}1657[#1657] 
+* Add experimental support for running the elastic-agent-shipper {agent-pull}1527[#1527] {agent-issue}219[#219]
+* Capture stdout/stderr of all subprocesses and adjust default log level to INFO for all components {agent-pull}1702[#1702] {agent-issue}221[#221]
```

Comment:
The wording has changed for this one:
`Collect logs from sub-processes via stdout and stderr and write them to a single, unified Elastic Agent log file.`